File: app/src/main/java/com/rungenes/firebasevisualsapplication/ModelClass.java
package com.rungenes.firebasevisualsapplication;

import com.google.firebase.database.Exclude;

public class ModelClass {

    String title;
    String image;
    String description;
    String uid;

    public ModelClass() {
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getImage() {
        return image;
    }

    public void setImage(String image) {
        this.image = image;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

@Exclude
    public String getUid() {
        return uid;
    }

    @Exclude
    public void setUid(String uid) {
        this.uid = uid;
    }
}

File: app/src/main/java/com/rungenes/firebasevisualsapplication/ModelClass.kt
package com.rungenes.firebasevisualsapplication

import com.google.firebase.database.Exclude

class ModelClass {
    var title: String? = null
    var image: String? = null
    var description: String? = null

    @get:Exclude
    @set:Exclude
    var uid: String? = null
}